ts-apps/webview/src/routes/+page.svelte

```diff
@@ -98,6 +100,11 @@
     }
   }
 
+  /** So there's space for the fn name */
+  .slightly-shorter-than-full-viewport-height {
+    height: 96svh;
+  }
```

Comment:
is that how you do it? shouldn't the fn name take its space and then what comes below fill the remaining space?

Reply:
The fn name does take its space, you're right. But if I recall correctly, if the height of the div containing the flow is set to some svh, call it *x*, then the height of that div will still be *x* regardless of the height of the fn name element. It's just that the place where the div containing the flow begins that will change when the height of the fn name changes.

So the thought is, if one wants everything to fit nicely on first load in the vscode webview, as I do, then it makes sense to make height of the div containing the flow a bit smaller.

I was experimenting quite fast with this just now and I definitely don't have all the details of the various css height rendering algorithms down, so I might very well be wrong about the details.